spec/services/get_or_create_user_spec.rb

```diff
@@ -2,13 +2,13 @@
 
 describe GetOrCreateUser do
   context '#call' do
-    let(:params) { {email: 'barf@eagle5.com', name: 'Barf'} } 
+    let(:params) { {email: 'barf@eagle5.com', nickname: 'Barf'} }
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.